rust/sbp/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "sbp"
+version = "0.1.0"
+authors = ["Gareth McMullin <gareth@swiftnav.com>"]
+description = "Rust native implementation of SBP (Swift Binary Protocol) for communicating with devices made by Swift Navigation"
+
+[dependencies]
+byteorder = "1.2.1"
+crc16 = "*"
+nom = "5.0.0-beta2"
```

Comment:
Do we really need a beta version of nom?

Reply:
Not any more. Apparently version 5.0 has been released since I started work on this :)

Reply:
Ok, let's fix this up and merge.